docs-source/usersguide/application/02_running_sims.rst

```diff
@@ -593,6 +593,40 @@ such as :file:`charmm36/water.xml`, which specifies the default CHARMM water mod
 The converted parameter sets come from the `CHARMM36 July 2017 update <http://mackerell.umaryland.edu/charmm_ff.shtml>`_
 and were converted using the `openmm-forcefields <https://github.com/choderalab/openmm-forcefields>`_ package and `parmed <https://github.com/parmed/parmed>`_.
 
+Implicit Solvent
+----------------
+
+The Amber and CHARMM force fields described above can be used with any of the Generalized
+Born implicit solvent models from AMBER.  To use them, include an extra file when
+creating the ForceField.
+
+.. tabularcolumns:: |l|L|
+
+======================  ==================================================================================================================================
+File                    Implicit Solvent Model
+======================  ==================================================================================================================================
+:file:`implicit/hct`    Hawkins-Cramer-Truhlar GBSA model\ :cite:`Hawkins1995` (corresponds to igb=1 in AMBER)
+:file:`implicit/obc1`   Onufriev-Bashford-Case GBSA model\ :cite:`Onufriev2004` using the GB\ :sup:`OBC`\ I parameters (corresponds to igb=2 in AMBER).
+:file:`implicit/obc2`   Onufriev-Bashford-Case GBSA model\ :cite:`Onufriev2004` using the GB\ :sup:`OBC`\ II parameters (corresponds to igb=5 in AMBER).
+:file:`implicit/gbn`    GBn solvation model\ :cite:`Mongan2007` (corresponds to igb=7 in AMBER).
+:file:`implicit/gbn2`   GBn2 solvation model\ :cite:`Nguyen2013` (corresponds to igb=8 in AMBER).
+======================  ==================================================================================================================================
+
+You can further control the solvation model in a few ways.  First, you can
+specify the dielectric constants to use for the solute and solvent:
+::
+
+    system = forcefield.createSystem(topology, soluteDielectric=1.0, solventDielectric=80.0)
+
+If they are not specified, the solute and solvent dielectrics default to 1.0 and
+78.5, respectively.
+
+You also can model the effect of a non-zero salt concentration by specifying the
+Debye-Huckel screening parameter\ :cite:`Srinivasan1999`:
```

Comment:
Can we give any guidance on how one might identify the appropriate kappa from a more familiar ionic strength?

Reply:
Good question.  Here's the code in the Amber file parser to compute kappa:

https://github.com/openmm/openmm/blob/ad113a0cb37991a2de67a08026cf3b91616bafbe/wrappers/python/openmm/app/amberprmtopfile.py#L267-L274

All of that seems straightforward, except for the factor of 0.73 "to account for ion exclusions".  I have no idea what that is talking about.  I looked in the Amber manual, but couldn't find any description of it there.

Reply:
I wonder if it would make sense to accept either `implicitSolventSaltConc` or `implicitSolventKappa` as arguments? I imagine the former will be much more convenient!

Reply:
> All of that seems straightforward, except for the factor of 0.73 "to account for ion exclusions". I have no idea what that is talking about. I looked in the Amber manual, but couldn't find any description of it there.

Caution, *highly* precise language ahead:

I think I recall seeing this in one of the original GB papers that introduced the approximation.  If I recall correctly, it's an empirical factor that improves performance of the model, explained as accounting for ions actually taking up space (and not being point-charges).  The comment is equally vague in sander:

```Fortran
      !       ---scale kappa by 0.73 to account(?) for lack of ion exclusions:

      kappa = 0.73d0* kappa
```

Reply:
Good to know I had a reason for being confused!  The comment in the Amber code is actually much more informative.  The (?) tells you that the person who wrote it didn't understand it either. :)  I guess I'll just roll it into the prefactor and give a formula with a magic constant in the manual.